cmd/brew-setup-nginx-conf.rb

```diff
@@ -110,6 +110,7 @@
 end
 
 server_base_path = "/usr/local/etc/nginx/servers"
+system "sudo mkdir -p #{server_base_path}"
```

Comment:
This path should be writeable without `sudo`, since we're doing a sudoless `ln` below. In fact, if we create the path with `sudo` then the `ln` below might fail!
```suggestion
system "mkdir -p '#{server_base_path}'"
```

Reply:
oh good catch! I was meaning to do it without sudo and then copy/paste got me.